pkg/api/validation/validation.go

```diff
@@ -1159,6 +1175,14 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 			allErrs = append(allErrs, validateVsphereVolumeSource(pv.Spec.VsphereVolume, specPath.Child("vsphereVolume"))...)
 		}
 	}
+	if pv.Spec.PhotonPersistentDisk != nil {
```

Comment:
It's not 100% duplicated though... one is using fldPath and the other is using specPath. And I am trying to be consistent with the context code. 
Maybe a separate commit just for code-cleanup to shorten all the duplicated code in this block would be a better solution?